themes/color-black.yaml

```diff
@@ -13,8 +13,10 @@ global:
     transparent:                [0.900,0.900,0.900,0.250]
     route_line:                 [0.000,0.500,1.000]
 
-styles:
-    mapzen_icon_library:
-        # To make Refill's colorize-able icons play nice with other styles
-        # restate the mix so the order of import matters
-        mix: colorized-icons
+    # To make Refill's colorize-able icons play nice with other styles
+    # we re-state the global here in the Refill color theme to use colorized_icons.
+    # This allows the Refill color theme to import *after* say Walkabout icons
+    # and then colorize Walkabout's icons. But when imported before Walkabout icons
+    # then Walkabout icons would not be colorized as Walkabout icons re-set the
+    # global to "" (null).
```

Comment:
Ah, I suppose there it still a way that import order can impact this behavior! Good that there's a comment about it.

Reply:
yeah which value of `global.sdk_icon_color_style` wins is dependent on whats the last import which specified this value. So yeah, its still order dependent.